test/unit/base_test.rb

```diff
@@ -34,10 +34,10 @@ def test_process_response_returns_hash
     end
 
     def test_process_response_returns_string
-      response = OpenStruct.new(:body => 'true')
+      response = OpenStruct.new(:body => 'Test body')
       data = @my_runcible.process_response(response)
 
-      assert_equal 'true', data.body
+      assert_equal 'Test body', data.body
```

Comment:
@daviddavis was trying to fix this here:  https://github.com/Katello/runcible/pull/180

I think this okay for a temporary fix